guide/guide/src/main/scala/io/udash/web/guide/StatesToViewFactoryDef.scala

```diff
@@ -42,7 +42,7 @@ class StatesToViewFactoryDef extends ViewFactoryRegistry[RoutingState] {
 
       case RestState => MarkdownPageViewFactory[RestState.type]()(Context.serverRpc.pages)
 
-      case I18NExtState => MarkdownPageViewFactory[I18NExtState.type]()(Context.serverRpc.pages)
+      case I18NExtState => I18NExtViewFactory
```

Comment:
I understand this is only temporary, until we figure out the `I18n` fiddles?

Reply:
We cannot use them until new RPC is in.